routes/api/comments/index.js

```diff
@@ -22,34 +26,59 @@ router.get('/:comment_id', (req, res, next) => {
 
 router.post('/', (req, res, next) => {
   let comment  = new Comment({
-    body: req.query.body,
-    author_id: req.query.author_id,
-    asset_id: req.query.asset_id,
-    parent_id: req.query.parent_id,
-    status: req.query.status
+    body: req.body.body,
+    author_id: req.body.author_id,
+    asset_id: req.body.asset_id,
+    parent_id: req.body.parent_id,
+    status: req.body.status
```

Comment:
if you want to, you can use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to pull the properties off `req.body` to make for less repetition.

`const {body, author_id, asset_id, parent_id, status} = req.body;`

and then you would just use the variables as normal.

`let comment = new Comment({body, author_id, asset_id, parent_id, status});`